content/tutorials/tags.json

```diff
@@ -0,0 +1,150 @@
+{
```

Comment:
How would we add Hornet or Bee tutorials? Integration services? Tools?
Would it make sense to add another dropdown with projects?

Reply:
Ok. Should we add the tags in this PR?
I guess in the stardust testing phase we may need to add tags we see needed anyway. Wdyt?

Reply:
Sure, I'll add them 👍